### DISCUSSION TITLE
Есть ли возможность обновить время вручную во время ввода в поле input?

### DISCUSSION THREAD

**Discussion by albafox** [t1m0n/air-datepicker#598] [Q&A] [answered]
На данный момент я нашёл как вручную можно обновить дату, например это работет через selectedDates или viewDate, и я думал, что время можно тоже обновить таким же образом, через например startDate. Но почему-то время не обновляется, может я что-то не так делаю? Или пока просто нет такого функционала?

Вот пример кода, это пока тестовый образец, но просто чтобы было понятно о чём речь. Спасибо.
```javascript
// Update datepicker when input is changed manually
$(input).on('input', function() {
    let valDate = input.value.split(' ')[0];
    let valTime = input.value.split(' ')[1];
    console.log('valDate: ' + valDate + ' | valTime: ' + valTime);

    if (isDateValid(input.value) && isValidDateString(valDate) && isValidTimeString(valTime)) {
        let date = new Date(input.value);
        let viewDate = dp.viewDate;

        viewDate.setHours(date.getHours(), date.getMinutes(), 0);
        console.log('date: ' + date);
        console.log('viewDate: ' + viewDate);

        dp.update({
            selectedDates: date,
            startDate: date,
            viewDate,
        });
    }
});
```

**Comment by t1m0n** [comment, marked as answer]
Можно использовать метод [`selectDate`](https://air-datepicker.com/ru/methods?scrollTo=selectDate) с параметром `updateTime: true`

```javascript
dp.selectDate('2024-03-19T10:30', {updateTime: true})
```

**Reply by albafox** [reply]
Спасибо огромное! Как раз то что было нужно!
